package.json

```diff
@@ -185,6 +185,7 @@
 		"eslint-import-resolver-node": "0.3.4",
 		"eslint-plugin-eslint-comments": "3.1.2",
 		"eslint-plugin-import": "2.25.2",
+		"eslint-plugin-jest": "27.2.1",
```

Comment:
We're now relying on it directly, so adding it as a dependency. Using the latest version.